### ISSUE TITLE
feature request: alt text via MSC3552

### ISSUE THREAD

**Issue by lavenderdotpet** [SableClient/Sable#1655]
https://github.com/matrix-org/matrix-spec-proposals/pull/3552

any chance of ui for alt text being support via MSC3552

description taken from the spec proposal
``
Optional - An m.alt_text block to represent alternative/descriptive text for the image. This is used as an accessibility feature, and per the block's definition above should only contain a plain text representation at the moment. Clients are encouraged to assume there is no alt text if no plain text representations are present. For clarity, this value would be supplied to the alt attribute of an img node in HTML.
}
``

moved to new issue due to a dif msc actually being for alt text ass mentioned here
https://github.com/SableClient/Sable/issues/1652#issuecomment-5199219978

**Comment by 7w1** [comment]
Issue with MSC3552 and the related extensible event PRs is that they require a new room version (aka not getting implemented in any reasonable time frame).

[MSC4460](https://github.com/matrix-org/matrix-spec-proposals/pull/4460) seems like a potential solution to that, allowing events to be used without a new room version.

Of course the caveat here is that atm that alt text would only be supported in Sable, but its better than nothing.
